jest.config.js

```diff
@@ -9,7 +9,8 @@ const requiresPrettierInternals = [
 ];
 
 const semver = require("semver");
-const isOldNode = semver.parse(process.version).major <= 4;
+const version = semver.parse(process.version);
+const isOldNode = version && version.major <= 4;
```

Comment:
Let's focus on src/**/*.js. Build scripts and the like can be taken care of later.

Reply:
Reverted this update & reverted other updates related to jest.config.js